### PR TITLE
deploy file

### PR DESCRIPTION
**Pull request by YagorlitskiyVyacheslav**

### CODE REVIEW

**Review by Artem-Rysich — COMMENTED** [review]
Like Good_)

File: .github/workflows/deploy.yml [hunk 1]
```diff
@@ -2,7 +2,7 @@ name: Build and deploy to GitHub Pages
 
 on:
   push:
-    branches: [main]
+    branches: [master]
```
Comment (Artem-Rysich) [inline .github/workflows/deploy.yml:5]: Lets discuses

**Review by Artem-Rysich — CHANGES_REQUESTED** [review]

File: .github/workflows/deploy.yml (hunk 1 above)
Comment (YagorlitskiyVyacheslav) [reply .github/workflows/deploy.yml:5]: ok

Comment (YagorlitskiyVyacheslav) [reply .github/workflows/deploy.yml:5]: I fixed it

**Review by Artem-Rysich — APPROVED** [review]

**Pull request merged**